src/config/graphical-session/gnome.md

```diff
@@ -2,27 +2,28 @@
 
 ## Pre-installation
 
+GNOME supports both Xorg and Wayland sessions. Follow [Wayland](./wayland.md),
+[Xorg](./xorg.md) sections to setup your preferred environment.
+
 Install the `dbus` package, ensure the `dbus` service is enabled, and reboot for
 the changes to take effect.
 
 ## Installation
 
-Install the `gnome` package for a GNOME environment which includes GNOME
-applications.
+Install the `gnome` package for a GNOME environment which includes the base
+GNOME desktop and a subset of GNOME applications. Additional applications are
+available via the `gnome-apps` package.
+
+A minimal GNOME environment can be created by installing the `gnome-core`
+package. (Note, however, that not all GNOME features may be present or
```

Comment:
I think we can remove the parentheses here, what do you think? Still leave the content, just remove the parentheses themselves.

Reply:
Removed.